jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jakarta-tests/src/test/java/org/eclipse/jetty/ee10/websocket/jakarta/tests/client/AnnotatedClientEndpointTest.java

```diff
@@ -78,7 +77,6 @@ public static class AnnotatedEndpointConfigurator extends ClientEndpointConfig.C
         @Override
         public void afterResponse(HandshakeResponse hr)
         {
-            hr.getHeaders().put("X-Test", Collections.singletonList("Extra"));
             super.afterResponse(hr);
```

Comment:
It was not tested that the header was set, because it was only set in this local map returned from getHeaders(), this is after the response has been received on the client. 

So these headers are now immutable.

I will leave the class because it is tested that a custom Configurator can be set, but I will remove the method override.

Reply:
Override was not removed?

Reply:
just forgot to push the changes